dev/src/main/java/org/eclipse/codewind/intellij/ui/tree/CodewindTreeNodeCellRenderer.java

```diff
@@ -192,10 +192,14 @@ private String getText(LocalConnection connection) {
                 // An older version is installed
                 return text + " [" + message("CodewindWrongVersionQualifier", status.getInstalledVersions()) + "] (" +
                         message("CodewindWrongVersionMsg", InstallUtil.getVersion());
-            } else if (status.isUnknown()) {
+            } else if (status.isError()) {
                 return text + " [" + message("CodewindErrorQualifier") + "] (" + message("CodewindErrorMsg") + ")";
+            } else if (status.isUnknown()) {
```

Comment:
Does this mean for an unknown status, we will simply show `Local` without any descriptive text to the right to indicate why it is unknown?  Previously, we tell the user to check the logs.

Reply:
@keithchong  Hi Keith, given this change, the only reason the status is unknown when the view is opened is that we are in the process of determining what the status is.  If an actual error occurs, the status will change to ERROR.

I have updated the code to show descriptive text in the view saying "Refreshing Codewind status..." when the status is unknown.